apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/rest/endpoint/DexController.java

```diff
@@ -603,11 +605,82 @@ public Response getUserFilledOrders(@Parameter(description = "Number of first N
+        responses = @ApiResponse(description = "List of user addresses", responseCode = "200",
+            content = @Content(mediaType = "application/json", schema = @Schema(implementation = String.class))))
+    public Response getAllUsers() {
+        List<String> addresses;
+        try {
+            addresses = service.getAllUsers();
+        } catch (AplException.ExecutiveProcessException e) {
+            return Response.status(Response.Status.BAD_REQUEST).build();
+        }
+        return Response.ok(addresses).build();
+    }
+
+    @GET
+    @Path("/eth/filledOrders")
+    @Produces(MediaType.APPLICATION_JSON)
+    @Operation(tags = {"dex"}, description = "Get all users filled orders on the smart contract. ",
+        responses = @ApiResponse(description = "List of user filled orders", responseCode = "200",
+            content = @Content(mediaType = "application/json", schema = @Schema(implementation = String.class))))
+    public Response getAllUsersFilledOrders() {
+
+        List<AddressEthDepositsInfo> addressEthDepositsInfos;
+        try {
+            addressEthDepositsInfos = service.getAllFilledOrders();
+        } catch (AplException.ExecutiveProcessException e) {
+            return Response.status(Response.Status.BAD_REQUEST).build();
+        }
+
+        return Response.ok(addressEthDepositsInfos).build();
+    }
+
+    @GET
+    @Path("/eth/expiredSwaps")
+    @Produces(MediaType.APPLICATION_JSON)
+    @Operation(tags = {"dex"}, description = "Get all users expired swaps on the smart contract. ",
+        responses = @ApiResponse(description = "List of user expired swaps", responseCode = "200",
+            content = @Content(mediaType = "application/json", schema = @Schema(implementation = String.class))))
+    public Response getAllUsersExpiredSwaps() {
+
+        List<AddressEthExpiredSwaps> addressEthExpiredSwaps;
+        try {
+            addressEthExpiredSwaps = service.getAllExpiredSwaps();
+        } catch (AplException.ExecutiveProcessException e) {
+            return Response.status(Response.Status.BAD_REQUEST).build();
+        }
+
+        return Response.ok(addressEthExpiredSwaps).build();
+    }
+
+
+    @GET
+    @Path("/eth/activeDeposits")
+    @Produces(MediaType.APPLICATION_JSON)
+    @Operation(tags = {"dex"}, description = "Get all users active deposits on the smart contract. ",
+        responses = @ApiResponse(description = "List of user active deposits", responseCode = "200",
+            content = @Content(mediaType = "application/json", schema = @Schema(implementation = String.class))))
+    public Response getAllUsersActiveDeposits() {
+
+        List<AddressEthDepositsInfo> addressEthExpiredSwaps;
+        try {
+            addressEthExpiredSwaps = service.getAllActiveDeposits();
+        } catch (AplException.ExecutiveProcessException e) {
+            return Response.status(Response.Status.BAD_REQUEST).build();
```

Comment:
You can build error message using ApiErrors and ResponseBuilder classes.

Reply:
It's inner functions for debugging, I don't think it make sense.